loops/loop49.go

```diff
@@ -0,0 +1,21 @@
+/* Matrix Addition */
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	m1 := [2][2]int{ {1, 2}, {3, 4}	}
+	m2 := [2][2]int{ {3, 1}, {4, 2} }
+	m3 := [2][2]int{}
+	fmt.Println(m1)
+	fmt.Println(m2)
+	for i:=0; i < 2; i++ {
+		for j:=0; j < 2; j++ {
+			m3[i][j] = m1[i][j] + m2[i][j]
+			fmt.Printf("%d ", m3[i][j])
```

Comment:
Received: 
4 3 
7 6 

Expected:
4 6 
7 3

Reply:
check  https://paste.centos.org/view/061b68b7

Reply:
In your paste you have 
`1       2               3       1`
`3       4               4       2`

But in the code you have
`{1, 2}, {3, 4}`
`{3, 1}, {4, 2}`